components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -323,6 +323,21 @@ public enum ErrorMessages {
                 "Application cannot be created for the sub-organization with id: %s."),
         ERROR_CODE_NO_PARENT_ORG("60079", "No parent organization.",
                 "No parent organization is available for the give organization id: %s."),
+        ERROR_CODE_DISCOVERY_CONFIG_DISABLED("60080", "Can't add organization discovery attribute.",
+                "The organization discovery configuration is disabled in the root organization with ID: %s."),
```

Comment:
```suggestion
                "The organization discovery configuration is disabled in the root organization with id: %s."),
```
to keep the consistency

Reply:
since we are also using `ID` in the messages, resolving the comments